### PR TITLE
feat: add button to opt-out of analytics to privacy policy page

### LINKED ISSUE #30: cookie banner + legal notice page
- We need a legal notice page + cookie banner a là https://www.dariah.eu/legal-notice
- the legal notice page should be linked from the footer banner
- the webhosting info should say campus is hosted on Netlify, not CNRS

### PR DESCRIPTION
**Pull request by stefanprobst**
this pr adds a button to the "Privacy Notice" page to manually opt out of Google Analytics.

closes #30

### CODE REVIEW

**Comment by VickyGarnett** [comment]
Hi Stefan - I'm happy to merge this, but it's telling me there is a conflict with the previous Consent Banner.  Is this something you can resolve?  Cheers - V

File: src/analytics/ConsentBanner.tsx [hunk 1]
```diff
@@ -19,23 +20,22 @@ export function ConsentBanner(props: ConsentBannerProps): JSX.Element {
...
-      >
-        {t('common.consent.reject')}
-      </button>
-      <button
-        className="px-3 py-1 text-sm font-medium bg-white rounded-full text-primary-600 focus:ring focus:ring-white focus:ring-offset-primary-600"
-        onClick={props.onAccept}
-      >
-        {t('common.consent.accept')}
-      </button>
+      <Button onClick={props.onDecline}>{t('common.consent.reject')}</Button>
```
Comment (stefanprobst) [inline src/analytics/ConsentBanner.tsx]:
```suggestion
      <Button onClick={props.onReject}>{t('common.consent.reject')}</Button>
```

**Comment by stefanprobst** [comment]
done! ready to merge

**Comment by VickyGarnett** [comment]
Job's a good'un!  Thanks Stefan :) - V

**Pull request merged**